ui/app/AppLayouts/Chat/ChatColumn/Message.qml

```diff
@@ -360,6 +346,31 @@ Item {
             source: contentType === Constants.stickerType ? ("https://ipfs.infura.io/ipfs/" + sticker) : ""
             visible: contentType === Constants.stickerType
         }
+
+        MouseArea {
+            anchors.fill: parent
+            cursorShape: chatText.hoveredLink ? Qt.PointingHandCursor : Qt.ArrowCursor
+            onClicked: {
+                let link = chatText.hoveredLink;
+                if(link.startsWith("#")){
+                    chatsModel.joinChat(link.substring(1), Constants.chatTypePublic);
+                    return;
+                }
+
+                if (link.startsWith('//')) {
+                  let pk = link.replace("//", "");
+                  profileClick(chatsModel.userNameOrAlias(pk), pk, chatsModel.generateIdenticon(pk), text)
+                  return;
+                }
+
+                Qt.openUrlExternally(link)
+            }
+            onPressAndHold: {
```

Comment:
I think this should be right-click since we are on desktop

Reply:
agree